pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeUtils.java

```diff
@@ -49,6 +50,8 @@
                                            Integer grpcPort,
                                            Long expectedHealthCheckInterval,
                                            String logConfigFile,
+                                           String secretsProviderClassName,
```

Comment:
Can some docs be added to what the secretsProviderConfig could or does contain. Should the classname just go in the secretsProviderConfig  so there is only one variable?